From 54ffd67b81a9ae36d7df16fdb9c1f262fc5ee446 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Wed, 30 Aug 2017 12:23:30 +0530
Subject: [PATCH 539/706] vfio fsl-mc: Use header files as per 4.13 kernel

dprc and mc-bus header files are changed in 4.13 from 4.11
VFIO driver is reworked to use those header files

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
(cherry picked from commit 69050090a41e42236c4728ac0b6f41fe2055765a)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/vfio/fsl-mc/vfio_fsl_mc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index 9dc32d273812..b997fb2f1069 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -20,8 +20,7 @@
 #include <linux/delay.h>
 
 #include "../../staging/fsl-mc/include/mc.h"
-#include "../../staging/fsl-mc/include/mc-bus.h"
-#include "../../staging/fsl-mc/include/mc-sys.h"
+#include "../../staging/fsl-mc/include/dprc.h"
 #include "../../staging/fsl-mc/bus/dprc-cmd.h"
 
 #include "vfio_fsl_mc_private.h"
-- 
2.17.1

